test/Microsoft.Extensions.Internal.Test/project.json

```diff
@@ -3,49 +3,25 @@
     "embed": {
       "include": "../../src/*.Sources/**/*.resx"
     },
-    "warningsAsErrors": true
+    "warningsAsErrors": true,
+    "compile": [
+      "../../src/Microsoft.Extensions.ClosedGenericMatcher.Sources/**/*.cs",
+      "../../src/Microsoft.Extensions.CopyOnWriteDictionary.Sources/**/*.cs",
+      "../../src/Microsoft.Extensions.DotnetToolDispatcher.Sources/**/*.cs",
+      "../../src/Microsoft.Extensions.HashCodeCombiner.Sources/**/*.cs",
+      "../../src/Microsoft.Extensions.PropertyActivator.Sources/**/*.cs",
+      "../../src/Microsoft.Extensions.PropertyHelper.Sources/**/*.cs",
+      "../../src/Microsoft.Extensions.SecurityHelper.Sources/**/*.cs",
+      "../../src/Microsoft.Extensions.TypeNameHelper.Sources/**/*.cs",
+      "../../src/Microsoft.Extensions.WebEncoders.Sources/**/*.cs"
+    ]
   },
   "dependencies": {
     "dotnet-test-xunit": "2.2.0-*",
     "Microsoft.DotNet.Cli.Utils": "1.0.0-preview2-1-003177",
-    "Microsoft.Extensions.ClosedGenericMatcher.Sources": {
-      "type": "build",
-      "version": "1.2.0-*"
-    },
-    "Microsoft.Extensions.CopyOnWriteDictionary.Sources": {
-      "type": "build",
-      "version": "1.2.0-*"
-    },
-    "Microsoft.Extensions.DotnetToolDispatcher.Sources": {
-      "type": "build",
-      "version": "1.2.0-*"
-    },
-    "Microsoft.Extensions.HashCodeCombiner.Sources": {
-      "type": "build",
-      "version": "1.2.0-*"
-    },
-    "Microsoft.Extensions.PropertyActivator.Sources": {
-      "type": "build",
-      "version": "1.2.0-*"
-    },
-    "Microsoft.Extensions.PropertyHelper.Sources": {
-      "type": "build",
-      "version": "1.2.0-*"
-    },
-    "Microsoft.Extensions.SecurityHelper.Sources": {
-      "type": "build",
-      "version": "1.2.0-*"
-    },
-    "Microsoft.Extensions.TypeNameHelper.Sources": {
-      "type": "build",
-      "version": "1.2.0-*"
-    },
-    "Microsoft.Extensions.WebEncoders.Sources": {
-      "type": "build",
-      "version": "1.2.0-*"
-    },
     "Moq": "4.6.36-*",
-    "xunit": "2.2.0-*"
+    "xunit": "2.2.0-*",
+    "Microsoft.Extensions.PlatformAbstractions": "1.2.0-*"
```

Comment:
The project.json for these apps includes these dependencies. We could preserve this behavior by putting it in the nuspec file, but we'd have to hard-code a dependency version since nuspec doesn't support floating dependency versions (-*). Instead, we'll need to update anyone using the .Sources package to include these additional dependencies.